gammapy/maps/region/geom.py

```diff
@@ -121,7 +128,12 @@ def _rectangle_bbox(self):
         for region_pix in regions_pix[1:]:
             bbox = bbox.union(region_pix.bounding_box)
 
-        rectangle_pix = bbox.to_region()
+        try:
+            rectangle_pix = bbox.to_region()
+        except ValueError:
+            rectangle_pix = RectanglePixelRegion(
+                center=PixCoord(*bbox.center[::-1]), width=1, height=1
```

Comment:
Why are the width and height set to 1 in this case?

Reply:
I think this up to us to define: is the width of the `RegionGeom` for a point sky region zero or the size of one pixel? I decided for the latter case, because this is basically how to treated technically.